### ISSUE TITLE
Run FBA tests on all BiGG models

### ISSUE THREAD

**Issue by matthiaskoenig** [shalinshah1993/SBSCL#28]
FBA simulations with all BiGG models should be performed and compared against reference solutions from cobrapy. 
BiGG models are not part of the repository, but must be downloaded for testing in CI or locally. Access is via environment variable.

**Comment by matthiaskoenig** [comment]
Most of this is implemented in
https://github.com/shalinshah1993/SBSCL/commit/cefc069f6000ffe7ff252a6049413a5f422d33e2

This issue depends on #24, i.e. only if FBA is working than it can be run on all BiGG models.

**Comment by shalinshah1993** [comment]
This issue was moved to draeger-lab/SBSCL#4
